leetcode/linked-list.md

```diff
@@ -0,0 +1,313 @@
+# linked-list
```

Comment:
ALL WRONG in commit message/nALL WRONG in commit message/nALL WRONG in commit message